packages/Input/src/index.tsx

```diff
@@ -43,17 +43,25 @@ const Input: FunctionComponent<IInputProps> = ({
   utils: { classNames, ErrorMessage },
   ...attrs
 }): JSX.Element => {
-  const [newValue, setNewValue] = useState<string | undefined>(value);
+  const isValidValue = value !== null && value !== undefined;
+
+  const [newValue, setNewValue] = useState<string | undefined>(
+    isValidValue ? value : ""
+  );
   const [newType, setType] = useState<string | undefined>(type);
+  const [newError, setError] = useState<boolean | undefined>(error);
 
-  useEffect(() => setNewValue(value), [value]);
+  useEffect(() => {
+    setError(isValidValue ? false : true);
```

Comment:
could we do it `setError(!isValidValue);`?

Reply:
Fixed